src/Libraries/Inscriptions/Envelope.ts

```diff
@@ -79,12 +79,13 @@ export class Envelope {
     }
   }
 
-  static fromTxinWitness(txid: string, txinwitness: string[]) {
+  static fromTxinWitness(txid: string, txinwitness: string[], currentEnvelopeIndex: number) {
```

Comment:
maybe `startingIndex` sounds better? so first envelop starts from `startingIndex = 0` and etc

Reply:
I think `currentEnvelopeIndex` is clearer, since this is the current index of the envelope within the Transaction